inference-engine/tests/functional/plugin/cpu/single_layer_tests/pooling.cpp

```diff
@@ -120,15 +121,104 @@ class PoolingLayerCPUTest : public testing::WithParamInterface<poolLayerCpuTestP
+
+        std::ostringstream results;
+        results << "IS=(";
+        results << CommonTestUtils::partialShape2str({inputShapes.first}) << ")_";
+        results << "TS=";
+        for (const auto& shape : inputShapes.second) {
+            results << CommonTestUtils::vec2str(shape) << "_";
+        }
+        results << "Prc=" << inPrc << "_";
+        results << "MaxPool_";
+        results << "K" << CommonTestUtils::vec2str(kernel) << "_";
+        results << "S" << CommonTestUtils::vec2str(stride) << "_";
+        results << "D" << CommonTestUtils::vec2str(dilation) << "_";
+        results << "PB" << CommonTestUtils::vec2str(padBegin) << "_";
+        results << "PE" << CommonTestUtils::vec2str(padEnd) << "_";
+        results << "Rounding=" << roundingType << "_";
+        results << "AutoPad=" << padType << "_";
+
+        results << CPUTestsBase::getTestCaseName(cpuParams);
+        results << CpuTestWithFusing::getTestCaseName(fusingParams);
+        return results.str();
+    }
+
+protected:
+    void SetUp() override {
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+
+        LayerTestsDefinitions::maxPoolV8SpecificParams basicParamsSet;
+        InputShape inputShapes;
+        ElementType inPrc;
+        CPUSpecificParams cpuParams;
+        fusingSpecificParams fusingParams;
+        std::tie(basicParamsSet, inputShapes, inPrc, cpuParams, fusingParams) = this->GetParam();
+
+        std::vector<size_t> kernel, stride, dilation;
+        std::vector<size_t> padBegin, padEnd;
+        ngraph::op::PadType padType;
+        ngraph::op::RoundingType roundingType;
+        std::tie(kernel, stride, dilation, padBegin, padEnd, roundingType, padType) = basicParamsSet;
+
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+        std::tie(postOpMgrPtr, fusedOps) = fusingParams;
+
+        if (selectedType.empty()) {
+            selectedType = getPrimitiveType();
+        }
+        selectedType = makeSelectedTypeStr(selectedType, inPrc);
+
+        init_input_shapes({inputShapes});
+
+        auto params = ngraph::builder::makeDynamicParams(inPrc, inputDynamicShapes);
+
+        std::shared_ptr<ngraph::Node> poolInput = params[0];
+
+        std::shared_ptr<ngraph::Node> pooling = ngraph::builder::makeMaxPoolingV8(params[0], stride, dilation, padBegin, padEnd,
+                                                                                  kernel, roundingType, padType);
+
+        ngraph::ResultVector results{std::make_shared<ngraph::opset3::Result>(pooling->output(0))};
+        function = std::make_shared<ngraph::Function>(results, params, "MaxPooling");
     }
 };
 
-TEST_P(PoolingLayerCPUTest, CompareWithRefs) {
+
+TEST_P(AvgPoolingLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    run();
+    //TODO: Need to rewrite tests because reference is used for some case
+    //CheckPluginRelatedResults(executableNetwork, "Pooling");
```

Comment:
Seems it was fixed.

Reply:
You're right but now some tests are executed via reference although `selected type` is `avx`. So tests are failed. 
I don't want to spent time to fix it and choose right shapes and attributes. I suggest to create follow-up task for it

Reply:
Without these tests we not sure about execution primitive. At least uncomment this row and add appropriate filter in skip_tests_config.cpp.

Reply:
Fixed all tests